variadics/src/lib.rs

```diff
@@ -274,6 +291,12 @@ where
     /// The un-referenced variadic. Each item will have one layer of shared references removed.
     ///
     /// The inverse of [`VariadicExt::AsRefVar`].
+    ///
+    /// ```rust
+    /// # use variadics::*;
+    /// let un_ref: <var_type!(&u32, &String, &bool) as RefVariadic>::UnRef =
```

Comment:
Is there no way to make this a method call so it rhymes with the other things here?